README.md

```diff
@@ -9,11 +9,37 @@ standalone tracing and observability platform, Traceable created this open sourc
 project for the software development community to use in their applications.
 
 ---
+## Quick-start
+If you want to see Hypertrace in action, you can quickly start Hypertrace via Docker.
 
+```
+git clone https://github.com/hypertrace/hypertrace.git
+cd hypertrace
+docker-compose -f docker/docker-compose.yml up
+```
+This will start all services required for Hypertrace. Once you see the service `Hypertrace-UI` start, you can visit Hypertrace UI at http://localhost:2020 . 
 
-### To start using Hypertrace
-- [Join the Hypertrace Workspace](https://www.hypertrace.org/get-started) on Slack to chat with other Hypertrace users.
-- You will be invited to a private channel where you can get the installer zip file along with instructions to install!
+If you have application instrumented to send traces to Zipkin or Jaeger, you are already covered with Hypertrace. Even if not you can run sample application using `docker-compose -f docker/docker-compose-zipkin-example.yml` or you can try Hypertrace with sample application by running
```

Comment:
Can we move this `or you can try Hypertrace with sample application by running` as a completely separate line?
as e.g
```
If you want to run both Hypetrace and sample application together, you can use below command
```